platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/builder/PlatformBuilder.java

```diff
@@ -659,6 +683,64 @@ public PlatformComponentBuilder buildComponentBuilder() {
         return new PlatformComponentBuilder(buildingBlocks);
     }
 
+    /**
+     * Creates a new roster from the bootstrap address book.
+     *
+     * @return a new roster
+     */
+    @NonNull
+    public Roster createRoster() {
```

Comment:
This new code seems like it would go in a utility class and having it here as public methods breaks the `Builder` pattern of the `PlatformBuilder`.  These methods on the utility class would be called in a `build()` call, maybe the `buildComponentBuilder`?  

If we're performing inversion of control, then the conversion of the address book to a roster happens completely before and outside of the `PlatformBuilder`?

Reply:
IOC is what the `PlatformBuilder.withBootstrapRoster` is for, as are other methods called from `Hedera` class so I think that bit is consistent. And in the case the platform itself wants to create the `Roster`, it can call this particular method directly. 
But you do have a point in putting it into a utility to preserve the builder pattern.